src/Lykke.Service.BlockchainCashoutPreconditionsCheck.Services/ValidationService.cs

```diff
@@ -45,123 +46,158 @@ public class ValidationService : IValidationService
+                                      false))
+                                {
+                                    errors.Add(ValidationError.Create(ValidationErrorType.FieldIsNotValid,
+                                        "Base Address should be part of destination address"));
+                                }
+
+                                var isBlockedBase = await _blackListService.IsBlockedAsync(
+                                    asset.BlockchainIntegrationLayerId,
+                                    destAddressParseResult.BaseAddress);
+
+                                if (isBlockedBase)
+                                    errors.Add(ValidationError.Create(ValidationErrorType.BlackListedAddress,
+                                        "Base Address is in the black list"));
+                            }
                         }
+                    }
 
-                        var isBlockedBase = await _blackListService.IsBlockedAsync(asset.BlockchainIntegrationLayerId,
-                            destAddressParseResult.BaseAddress);
+                    if (cashoutModel.ClientId != null)
+                    {
+                        var clientAddress = await _blockchainWalletsClient.TryGetAddressAsync(
+                            asset.BlockchainIntegrationLayerId,
+                            asset.BlockchainIntegrationLayerAssetId,
+                            cashoutModel.ClientId.Value);
 
-                        if (isBlockedBase)
-                            errors.Add(ValidationError.Create(ValidationErrorType.BlackListedAddress, "Base Address is in the black list"));
+                        if (string.Equals(clientAddress?.Address, cashoutModel.DestinationAddress,
+                            StringComparison.InvariantCultureIgnoreCase))
+                        {
+                            errors.Add(ValidationError.Create(ValidationErrorType.CashoutToSelfAddress,
+                                "Withdrawals to the deposit wallet owned by the customer himself prohibited"));
+                        }
                     }
                 }
             }
-
-            if (cashoutModel.ClientId != null)
+            catch (Exception ex)
             {
-                var clientAddress = await _blockchainWalletsClient.TryGetAddressAsync(asset.BlockchainIntegrationLayerId,
-                    asset.BlockchainIntegrationLayerAssetId,
-                    cashoutModel.ClientId.Value);
-
-                if (string.Equals(clientAddress?.Address, cashoutModel.DestinationAddress, StringComparison.InvariantCultureIgnoreCase))
-                {
-                    errors.Add(ValidationError.Create(ValidationErrorType.CashoutToSelfAddress, "Withdrawals to the deposit wallet owned by the customer himself prohibited"));
-                }
-
+                errors.Add(ValidationError.Create(ValidationErrorType.Error, ex.Message));
```

Comment:
You could use here a newly-implemented method
`ValidationError::Create(string value)`
with single parameter. If you don't, please, remove the method as unused one.